script/build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+##
+# Run the application and content builds. This script builds or fetches the
```

Comment:
Right, so this file is here so I don't have to change any of the interface with CI yet. We'll get there, but I wanted to roll that out separately since I think it'll be more involved.

Reply:
I tried to make the `npm` script do a simple `yarn build:webpack && yarn build:content`, but needed to pass the CLI args from the `yarn build` command to both, so that didn't work.

Reply:
Ah, I made this comment before you refactored a few things.  It makes more sense now!